provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/location/LocationController.java

```diff
@@ -170,7 +228,9 @@ public Location readByIdentifier(@PathVariable("publicId") String publicId) {
       locationWrapper.careSitesResponse(ppmsClient.careSitesByCity(city));
     } else if (parameters.get("address-state") != null) {
       String state = parameters.getFirst("address-state");
-      locationWrapper.careSitesResponse(ppmsClient.careSitesByState(state));
+      String mappedState = stateMap().get(state);
+      locationWrapper.careSitesResponse(
+          ppmsClient.careSitesByState(mappedState == null ? state : mappedState));
```

Comment:
You can do this check as part of the map lookup above with `You can do `stateMap().getOrDefault(state, state)``